### PR TITLE
Solve bug and remove daily summary

### PR DESCRIPTION
**Pull request by baristaGeek**
## Description
Clean-up to release 1.8

- Removed daily summary
- Uncommented the block of code that sends a message to paint prs per shas

## Type of change
- [x] Bug fix (non-breaking change which fixes an issue)  
- [ ] New feature (non-breaking change which adds functionality)  
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)  
- [ ] Documentation update  
- [x] Chore: cleanup/renaming, etc  
- [ ] RFC  

## Acceptance
- [x] I have read [the Contributing guidelines](CONTRIBUTING.md)
- [x] I have read the [Code of Conduct](CODE_OF_CONDUCT.md)

### CODE REVIEW

**Comment by baristaGeek** [comment]
@EstebanDalelR we still need to prompt the user with an invitation to purchase (eg: "Please upgrade Watermelon to have unlimited get code context usage") but I don't know how to implement it. 

[Here](https://github.com/watermelontools/watermelon/blob/main/pages/api/github/getIssuesByCommits.ts) I added a `return res.send({ error: "GitHub query limit reached" });` statement.

However for some reason, when I print the result of the promise in getPrsPerSHAs.ts I get "no repo". How do we fix this?

File: src/utils/vscode/getPRsToPaintPerSHAs.ts [hunk 1]
```diff
@@ -97,9 +96,10 @@ export default async function getPRsToPaintPerSHAs({
           state: issueData.state,
           draft: issueData.draft,
           number: issue.number,
-          comments: comments.map((comment: string) => {
-            return comment;
-          }),
+          comments: [],
```
Comment (EstebanDalelR) [inline src/utils/vscode/getPRsToPaintPerSHAs.ts:99]: Please use block comment

File: src/extension.ts [hunk 2]
```diff
@@ -209,6 +209,8 @@ export async function activate(context: vscode.ExtensionContext) {
           email: session.account.label,
           text: sortedPRs[0].title || parsedMessage,
         });
+        */
```
Comment (EstebanDalelR) [inline src/extension.ts:212]: This closes, but where does it open?

File: src/utils/vscode/getPRsToPaintPerSHAs.ts (hunk 1 above)
Comment (baristaGeek) [reply src/utils/vscode/getPRsToPaintPerSHAs.ts:99]: ok

File: src/extension.ts (hunk 2 above)
Comment (baristaGeek) [reply src/extension.ts:212]: On line 206

**Comment by baristaGeek** [comment]
Currently getting 401 for GitLab and "Token is invalid or not supported for this endpoint." for Bitbucket.

File: src/extension.ts (hunk 2 above)
Comment (EstebanDalelR) [reply src/extension.ts:212]: I will approve but I do not take responsibility for this possible bug

**Review by EstebanDalelR — APPROVED** [review]

**Pull request merged**